draft-ietf-keytrans-protocol.md

```diff
@@ -1805,6 +1772,91 @@ each label are not provided, given that these can be cached from the
 original Search or Update query.
 
 
+# Third Parties
+
+## Management
+
+With the Third-Party Management deployment mode, a third party is responsible
+for the majority of the work of storing and operating the Transparency Log,
+while the Service Operator serves mainly to enforce access control and
+authenticate the addition of new entries. All user queries specified in
+{{user-operations}} are initially sent by users directly to the Service Operator
+to be forwarded to the Third-Party Manager if they pass access control.
+
+While other operations are forwarded by the Service Operator unchanged,
+`UpdateRequest` structures are forwarded to the Third-Party Manager with the
+Service Operator's signature attached:
+
+~~~ tls-presentation
+struct {
+  UpdateRequest request;
+  opaque signature<0..2^16-1>;
+} ManagerUpdateRequest;
+~~~
+
+The signature is computed as described in {{update-format}}. The Service
+Operator MUST maintain its own records (independent of the Third-Party Manager)
+for the greatest version of each label for the purpose of producing this
+signature.
+
+## Auditing
+
+With the Third-party Auditing deployment mode, the Service Operator obtains
+signatures from a lightweight Third-Party Auditor attesting to the fact that the
```

Comment:
```suggestion
signatures from a Third-Party Auditor attesting to the fact that the
```

Partially personal preference. "Lightweight" is such an overused term and conveys little meaning. But also: I don't think auditors will be particularly lightweight, no? They may need to perform a lot of computations to check tree construction.